client/app/lib/components/contentModal/styl/contentModal.styl

```diff
@@ -275,11 +275,20 @@ main.main-container
     borderBox()
 
     .markdown-content
-      margin-top            10px
-      max-width             450px
-
-      pre code
-        overflow            auto
+      /* margin-top            10px */
+      /* max-width             450px */
+      padding                   10px 30px
```

Comment:
please take a another look to provided pdf, the spacings are not the same around the code block.